src/tasks/create-first-git-commit.ts

```diff
@@ -13,7 +13,9 @@ export async function createFirstGitCommit(targetDir: string, options: Options)
     if (options.solidityFramework === SOLIDITY_FRAMEWORKS.FOUNDRY) {
       const foundryWorkSpacePath = path.resolve(targetDir, "packages", SOLIDITY_FRAMEWORKS.FOUNDRY);
       // forge install foundry libraries
-      await execa("forge", ["install", ...foundryLibraries], { cwd: foundryWorkSpacePath });
+      await execa("forge", ["install", ...foundryLibraries, "--no-commit"], { cwd: foundryWorkSpacePath });
+      await execa("git", ["add", "-A"], { cwd: targetDir });
+      await execa("git", ["commit", "--amend", "--no-edit"], { cwd: targetDir });
```

Comment:
Ohh btw forgot to mention this, we are doing it because earlier(currently in `npx create-eth@latest -s foundry`) we create 3 commit (initial commit + install oz + install solidityBytes package) 

The reason behing last 2 commits is because `forge install` pulls the submodule and creates a new commit. 

Hence passed `--no-commit` to forge and put all the changes in initial commit using git ammend